libraries/Network/src/NetworkUdp.cpp

```diff
@@ -297,6 +297,13 @@ int NetworkUDP::parsePacket() {
   struct sockaddr_storage si_other_storage;  // enough storage for v4 and v6
   socklen_t slen = sizeof(sockaddr_storage);
   int len;
```

Comment:
here `len` should be set to `0`, else it might not be initialized and might not be `0` for the next check

Reply:
Done! I've clearly spent too much time in C# lately, where "out" variables are guaranteed to be assigned 😂